src/app.js

```diff
@@ -53,6 +53,7 @@ function setupConfiguration(configuration) {
       if (configFilter) {
         configFilter.enabled = true;
         configFilter.value = filter.value;
+        configFilter.operator = filter.operator;
```

Comment:
@dann-jobready it will set the `operator` field on every filter. Can we do it conditionally, only when the operator is present?